src/store/plugins/veeValidate.js

```diff
@@ -24,6 +24,7 @@ defineRule('min_value', (value, [arg]) => BigNumber(value).isGreaterThanOrEqualT
 defineRule('min_value_exclusive', (value, [arg]) => value && BigNumber(value).isGreaterThan(arg));
 defineRule('max_value', (value, [arg]) => value && BigNumber(value).isLessThanOrEqualTo(arg));
 defineRule('max_value_vault', (value, [arg]) => BigNumber(value).isLessThanOrEqualTo(arg));
+defineRule('max', (value, [maxLength]) => value && value.length <= maxLength);
```

Comment:
I think it would be better to name it `max_len` or something similar to precisely state what the rule is about